packages/venia-ui/lib/queries/createAccount.graphql

```diff
@@ -0,0 +1,19 @@
+mutation createAccount(
```

Comment:
Since this is the first mutation, I'm wondering if we shouldn't create a `lib/mutations/` folder. It's a bit weird to read the import:

```js
import CREATE_ACCOUNT_MUTATION from '../../queries/createAccount.graphql';
```

Reply:
A mutation is just a type of query. I don't think we should do this.

Reply:
Left more detail below.

Reply:
Maybe "a mutation is just a type of request"?

Reply:
[Queries and Mutations are separate, special types](https://graphql.org/learn/schema/#the-query-and-mutation-types) that serve as entry points for the respective operations.

The Magento GraphQL schema separates Queries from Mutations, here's GraphiQL:

![Screen Shot 2019-10-21 at 3 44 58 PM](https://user-images.githubusercontent.com/13182778/67241978-2777cb00-f41a-11e9-97ca-cdbf066908c2.png)

Whether or not the front end splits these things up into `queries` and `mutations` directories is a discussion worth having, but GraphQL and our Magento GraphQL schema does separate them, so I assumed we would want to too.

Is it the worst thing in the world if a `.graphql` file in the `queries/` directory contained `addConfigurableProductsToCart`? Probably not. 🤷‍♂